packages/abstractions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/kiota-abstractions",
-  "version": "1.0.0-preview.44",
```

Comment:
I've already bumped to 45 this morning in #1100. We'll need to rebase and bump all libraries one more preview